index.js

```diff
@@ -53,6 +53,7 @@ var config = {
 	animateInParallel: false,
 	appIcon: null,
 	pathToModule: '',
+	autoCleanup: true, // Auto cleanup
```

Comment:
We could release it as a new major version. Then I don't see a problem with a breaking change like this. I don't want people to run `npm update` and suddenly their `nw-notify` behaves differently.